### PR TITLE
test: add e2e test for com.docker.network.bridge.enable_icc option

### PR DESCRIPTION
**Pull request by swagatbora90**
Issue #, if available:

*Description of changes:*
Adds e2e test for enable_icc option
*Testing done:*
`sudo make test-e2e`

- [x] I've reviewed the guidance in CONTRIBUTING.md

#### License Acceptance

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: e2e/tests/network_create.go [hunk 1]
```diff
@@ -64,6 +65,17 @@ func NetworkCreate(opt *option.Option) {
 			}
 		}
 
+		cleanupNetworkWithHTTP := func(network string) func() {
+			return func() {
+				relativeUrl := fmt.Sprintf("/networks/%s", network)
+				apiUrl := client.ConvertToFinchUrl(version, relativeUrl)
+				req, err := http.NewRequest(http.MethodDelete, apiUrl, nil)
+				Expect(err).Should(BeNil())
```
Comment (pendo324) [inline e2e/tests/network_create.go:73]: minor style nit: use `Expect(err).ShouldNot(HaveOccurred())`

File: e2e/tests/network_create.go [hunk 2]
```diff
@@ -172,6 +184,55 @@ func NetworkCreate(opt *option.Option) {
 				Expect(httpResponse).Should(HaveHTTPStatus(http.StatusNotFound))
 			})
 		})
+
+		When("a network create request is made with newtork option com.docker.network.bridge.enable_icc set to false", func() {
```
Comment (pendo324) [inline e2e/tests/network_create.go:188]: typo: newtork => network

**Review by pendo324 — DISMISSED** [review]

File: e2e/tests/network_create.go (hunk 1 above)
Comment (swagatbora90) [reply e2e/tests/network_create.go]: fixed

**Review by pendo324 — APPROVED** [review]

File: e2e/tests/network_create.go [hunk 3]
```diff
@@ -172,6 +184,55 @@ func NetworkCreate(opt *option.Option) {
...
+				DeferCleanup(cleanupNetworkWithHTTP(testNetwork))
+
+				stdout := command.Stdout(opt, "network", "inspect", testNetwork)
+				Expect(stdout).To(ContainSubstring(`"finch.network.bridge.enable_icc.ipv4": "false"`))
+
+				// check iptables rules exists
+				iptOpt, _ := option.New([]string{"iptables"})
+				command.Run(iptOpt, "-C", "FINCH-ISOLATE-CHAIN",
+					"-i", testBridge, "-o", testBridge, "-j", "DROP")
+				time.Sleep(30 * time.Second)
```
Comment (austinvazquez) [inline e2e/tests/network_create.go:209]: Sync'd offline: to be removed.

**Review by austinvazquez — COMMENTED** [review]
LGTM, other than the one item.

File: e2e/tests/network_create.go (hunk 3 above)
Comment (swagatbora90) [reply e2e/tests/network_create.go:209]: Good catch. leftover from an earlier local test.

**Review by pendo324 — APPROVED** [review]

**Review by austinvazquez — APPROVED** [review]

**Pull request merged**